portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/databasepartitioning/DatabasePartitioning.testcase

```diff
@@ -197,6 +197,7 @@ definition {
 	@priority = 4
 	test CanAddCompanyWithHeadlessAPI {
 		property custom.properties = "virtual.hosts.valid.hosts=localhost,127.0.0.1,www.baker.com";
+		property database.auto.upgrade.enabled = "true";
```

Comment:
Why adding `database.auto.upgrade.enabled` on that db partition test?

I believe you got confused and thought it was `CanScheduleJobInVariousCompaniesWhenAutoUpgradeIsEnabled`, that was changed in the previous commit